layouts/partials/partner-form/form.html

```diff
@@ -24,9 +24,9 @@ <h2>Great! Let's get started.</h2>
     </div>
 
     <div style="display:none" class="partner-form-last-name step active">
-      <label class="label">Your Last Name</label>
+      <label class="label">Your Last Name*</label>
```

Comment:
why not bind label with input?

Reply:
Sorry I'm not sure what you mean by this, if this helps for this section I was copying the visual styling of the imbedded form we were using previously

Reply:
@mbouzi i have added more detailed comment here https://github.com/storj/tardigrade.io/pull/111#discussion_r450904514